internal/restic/node.go

```diff
@@ -719,12 +721,7 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	allowExtended, err := node.fillGenericAttributes(path, fi, stat)
 	if allowExtended {
 		// Skip processing ExtendedAttributes if allowExtended is false.
-		errEx := node.fillExtendedAttributes(path)
-		if err == nil {
-			err = errEx
-		} else {
-			debug.Log("Error filling extended attributes for %v at %v : %v", node.Name, path, errEx)
-		}
```

Comment:
Error from fillExtendedAttributes was getting eaten up after this code was updated in https://github.com/restic/restic/pull/4611/files#diff-ed902b2edb54dfe5e342c38a7e7d122cacc6b1e84e956dfc8689b5efd7922fd4

Reply:
I can move this to the next PR for extended attrs if needed.